lib/chromedriver.js

```diff
@@ -16,11 +16,11 @@ const DEFAULT_HOST = '127.0.0.1';
 const DEFAULT_PORT = 9515;
 class Chromedriver extends events.EventEmitter {
   constructor (args = {}) {
-    const {host, port, executable, cmdArgs, adbPort, verbose, logPath} = args;
+    const {host, port, executable, cmdArgs, adb, verbose, logPath} = args;
     super();
     this.proxyHost = host || DEFAULT_HOST;
     this.proxyPort = port || DEFAULT_PORT;
-    this.adbPort = adbPort;
+    this.adb = adb;
```

Comment:
We only use it in one place (all chrome handling is done through `appium-android-driver`) so it will be easy to handle the breaking change.